metrics/density/fast_footprint.sh

```diff
@@ -0,0 +1,556 @@
+#!/bin/bash
+# Copyright (c) 2017-2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+#
+# A script to gather memory 'footprint' information as we launch more
+# and more containers
+# It allows configuration of a number of things:
+# - which container workload we run
+# - which container runtime we run with
+# - when do we terminate the test (cutoff points)
+#
+# There are a number of things we may wish to add to this script later:
+# - sanity check that the correct number of runtime components (qemu, shim etc.)
+#  are running at all times
+# - some post-processing scripts to generate stats and graphs
+#
+# The script gathers information about both user and kernel space consumption
+# Output is into a .json file, named using some of the config component names
+# (such as footprint-busybox.json)
+
+# Pull in some common, useful, items
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Note that all vars that can be set from outside the script (that is,
+# passed in the ENV), use the ':-' setting to allow being over-ridden
+
+# Default sleep for 3s to let containers come up and finish their
```

Comment:
3s? :-)

I'd be tempted to just say the following so we don't need to maintain the info in two places:

> Default sleep time in seconds to let ...